config/peerpods/podvm/podvm-builder.sh

```diff
@@ -288,6 +305,7 @@ aws)
   install_aws_deps
   ;;
 libvirt)
+  check_operator_built_or_prebuilt
```

Comment:
This function should be renamed to something like `set_podvm_image_type` or similar.
And it's better to call this before the case statement.

Reply:
incorporated..